drivers/hmis/app/graphql/types/hmis_schema/ce_referral.rb

```diff
@@ -43,9 +43,10 @@ def self.authorized?(object, ctx)
     summary_field :custom_status, HmisSchema::CeCustomReferralStatus, null: true
     summary_field :client_id, ID, null: false
     summary_field :client_name, String, null: true, description: 'The name of the referred client. Always available to those who can view the full referral, even without full client record access.'
-    # Special case: Client is a "summary field" because it doesn't require referral visibility, but resolving it does require permission to view the client record.
```

Comment:
Removing unrelated - I think this comment was outdated, since `client` is _not_ a summary-level field.